modules/va_notify/app/controllers/va_notify/callbacks_controller.rb

```diff
@@ -18,6 +18,7 @@ def create
       if (notification = VANotify::Notification.find_by(notification_id: notification_id))
         Rails.logger.info("va_notify callbacks - Updating notification #{notification.id}")
         notification.update(notification_params)
+        VANotify::DefaultCallback.new(notification).call
```

Comment:
@nathanbwright I did opt for creating an instance and making `call` an instance method.